example_nativeapm.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+  <title>Native APM Example Page</title>
+</head>
+
+<body>
+  <div id="nativeapm-container" style="width: 350px"></div>
+  <script src="./scripts/processout.js"></script>
+  <script>
+    document.addEventListener("DOMContentLoaded", function () {
+      // Create a new ProcessOut client
+      var client = new ProcessOut.ProcessOut("proj_QHeiFsmPiaadVizlKTuJrTavLkqVImQ3");
+
+      // Create a new NativeAPM instance
+      var nativeApm = client.setupNativeApm({ invoiceId: 'iv_MoGID2FQe0YO7M1xwFwnn2Y36IPN8bHF', gatewayConfigurationId: 'gway_conf_2puMFv3iwrPSLqslIfVcvzhFWZzmTHTy' });
```

Comment:
If you want to test it with Processout.js you need to enter valid invoiceId and gatewayConfigurationId